local-cli/cliEntry.js

```diff
@@ -137,7 +137,7 @@ function run() {
 
   commander.parse(process.argv);
 
-  const isValidCommand = commands.find(cmd => cmd.name === process.argv[2]);
+  const isValidCommand = commands.find(cmd => cmd.name.split(" ")[0] === process.argv[2]);
```

Comment:
Which PR? I don't think https://github.com/facebook/react-native/pull/9173/files is related, it only fixes showing the help help. When you run the command `react-native link react-native-plugin`, you would still see the warning because `isValidCommand` fails to find a command with a name that matches.

I noticed you added "needs-revision", what exactly needs revision? Also, given that this is not the main point of this PR, I would appreciate it if you could comment on the addition of the pre/post unlink hooks :).